### PR TITLE
Simple cron program

### LINKED ISSUE #14: Set up Cloudflare cron trigger
In order to send  in the official ACM at CSUF Discord server, our HTTP server exposes an endpoint `POST https://lc-dailies.deno.dev/webhook/:token` which when triggered, a webhook detailing Leetcode's daily challenge question is delivered to the `📚algo-chat` text channel.

### Literature
- <https://developers.cloudflare.com/workers/examples/cron-trigger/>
- <https://developers.cloudflare.com/workers/configuration/cron-triggers/>
- <https://developers.cloudflare.com/workers/wrangler/configuration/>
- <https://denoflare.dev/guides/serve>

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
Resolves #14.

Runs every day [`0 0 * * *`](https://crontab.guru/#0_0_*_*_*).

### Changes

- New module worker `cf/dailies/dailies.ts`  (the file deployed to Cloudflare)
- New tasks `cf:serve`, `cf:push`
- New [Denoflare CLI](https://denoflare.dev) helper `lib/denoflare/denoflare.ts`

### CODE REVIEW

File: cloudflare/deploy.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        accountId: CF_ACCOUNT_ID,
+        apiToken: CF_API_TOKEN,
+      },
+    },
+  };
+  await Deno.writeTextFile(
+    DENOFLARE_CONFIG_FILENAME,
+    JSON.stringify(config),
+  );
+  console.log({ config });
```
Comment (EthanThatOneKid) [inline cloudflare/deploy.ts:41]: TODO: Remove irrelevant console logs.

File: deno.jsonc [hunk 2]
```diff
@@ -6,7 +6,8 @@
     "all": "deno task udd && deno lint && deno fmt",
     "test": "deno test --unstable",
     "start": "deno run -A --unstable main.ts",
-    "ngrok": "ngrok http 8080"
+    "ngrok": "ngrok http 8080",
+    "cf:deploy": "deno run -A cloudflare/deploy.ts"
```
Comment (EthanThatOneKid) [inline deno.jsonc:10]: TODO: Add task [`cf:serve`](https://denoflare.dev/guides/serve).

File: cloudflare/deploy.ts (hunk 1 above)
Comment (EthanThatOneKid) [reply cloudflare/deploy.ts]: Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/15/commits/d0678303b18a37b52f435ee3a3b5b63a327ee681#diff-854d7a8cbbd35474842f6c0c52644fd53faef44b2baccd0988b78a3b2c16a4ecL41>.

File: deno.jsonc (hunk 2 above)
Comment (EthanThatOneKid) [reply deno.jsonc]: Done in <https://github.com/acmcsufoss/lc-dailies/pull/15/commits/bf84e1781195c2990d64c6f940c4fd23489284e8>.

File: .env.example [hunk 3]
```diff
@@ -5,3 +5,6 @@ DISCORD_TOKEN="❤"
 DISCORD_PUBLIC_KEY="❤"
 DISCORD_WEBHOOK_URL="❤"
 DISCORD_CHANNEL_ID="❤"
+
+CF_ACCOUNT_ID="🧡"
+CF_API_TOKEN="🧡"
```
Comment (EthanThatOneKid) [inline .env.example:10]: Not required by core LC-Dailies program.

Comment (EthanThatOneKid) [reply .env.example]: Done in <https://github.com/acmcsufoss/lc-dailies/pull/15/commits/ffd3a3e189f56a573b7c270947ba239ddbef61df>.

**Comment by EthanThatOneKid** [comment]
Related: #17.

**Review by karnikaavelumani — APPROVED** [review]
Thank you for your hard work. Hoping this works!

**Pull request merged**